pywbem/mof_compiler.py

```diff
@@ -636,17 +636,12 @@ def p_mp_createInstance(p):
                     if p.parser.verbose:
                         p.parser.log(
                             _format("ModifyInstance not supported. "
-                                    "Deleting instance of {0!A}: {1}",
+                                    "Create instance of {0!A}: {1} failed",
                                     inst.classname, inst.path))
```

Comment:
I find this log message slightly confusing.
Plus, in the old code, this log message introduced that we are now deleting the instance. If we just want to log that ModifyInstance failed, we should do that for all status codes, and I guess that will be done by whoever handles the exception that is raised.

Bottom line, can we not remove the logging at this point? That would remove lines 635 to 640.

Reply:
Sounds OK to me.  DONE